packages/network-controller/src/index.ts

```diff
@@ -1,4 +1,39 @@
-export * from './NetworkController';
+export type {
+  Block,
+  NetworkMetadata,
+  NetworkConfiguration,
+  BuiltInNetworkClientId,
+  CustomNetworkClientId,
+  NetworkClientId,
+  NetworksMetadata,
+  NetworkState,
+  BlockTrackerProxy,
+  ProviderProxy,
```

Comment:
Could these 2 be exported as well?  Otherwise I end up re-creating them in the client.  Useful for representing a network before it's been added.

```suggestion
  ProviderProxy,
  AddNetworkFields,
  UpdateNetworkFields,
```

Reply:
Yup! Done.